Kevin Rockwell Assignment 1

kevinrockwell opened the pull request.

kevinrockwell committed:
Add hello world program

kevinrockwell committed:
Add Chapter 1 Assignment

- Create teams dictionary with data for 5 teams

kevinrockwell committed:
Change teams dict to store comp locations in list

- Makes it easier to access

kevinrockwell committed:
Add input from terminal

- Print valid teams and attributes
- Take inputs (case insensitive) and check to see if they are valid
-  Replace spaces in attribute input with underscores

kevinrockwell committed:
Add output

Ethan-Talbert reviewed (commented):
Above and beyond the requirements, good code.

kevinrockwell committed:
Make teams dictionary use integer keys

kevinrockwell committed:
Revert "Change teams dict to store comp locations in list"

makes competition location a dict again
This reverts commit b74cab82af7ea91d663993dcf2a0d92d254de287.

kevinrockwell committed:
Make output work for competiton locations

kevinrockwell committed:
Add Ch 2 assignment

kevinrockwell committed:
Prevent already added teams from being readded

kevinrockwell committed:
Make modify prompt for attributes

kevinrockwell committed:
Add basic input functions and general program structure

kevinrockwell committed:
Fix input functions, rename get_int to get_positive_int

- Rename makes clear what function actually does

kevinrockwell committed:
Add get_bool function

kevinrockwell committed:
Add attributes list, tweak wording in get_bool

- Now asks for yes/no instead of true/false

kevinrockwell committed:
Add get_comps and finish add team

kevinrockwell committed:
Implement team deletion

kevinrockwell committed:
Implement list teams (stolen from Ch 2)

kevinrockwell committed:
Implement basic search

- Only matches starts of (and complete) team number because team
  Name was not stored in the database

kevinrockwell committed:
Implement exit

kevinrockwell committed:
Implement modify, fix some stuff

kevinrockwell committed:
Add Ch 4 Assignment

kevinrockwell committed:
Add more comments to Ch 4 assignment
